systemtest/src/main/java/io/strimzi/systemtest/utils/HttpUtils.java

```diff
@@ -66,4 +76,90 @@ public static void waitUntilServiceHasSomeTraces(String baseURI, String... servi
             waitUntilServiceHasSomeTraces(baseURI, serviceName);
         }
     }
+
+    public static JsonObject generateHttpMessages(int messageCount) {
+        LOGGER.info("Creating {} records for Kafka Bridge", messageCount);
+        JsonArray records = new JsonArray();
+        JsonObject json = new JsonObject();
+        for (int i = 0; i < messageCount; i++) {
+            json.put("value", "msg_" + i);
+            records.add(json);
+        }
+        JsonObject root = new JsonObject();
+        root.put("records", records);
+        return root;
+    }
+
+    public static JsonObject sendHttpRequests(JsonObject records, String bridgeHost, int bridgePort, String topicName, WebClient client) throws InterruptedException, ExecutionException, TimeoutException {
```

Comment:
the method name sounds a little bit generic as "this method sends an HTTP request" and not "it's the method which implements the HTTP request for sending messages to the bridge". I would you something like `sendMessagesHttpRequest` or whatever you want which is less confusing.